public/index.html

```diff
@@ -0,0 +1,150 @@
+<html class="loader">
+
+<head>
+    <meta charset="UTF-8">
+    <title>DCML Reductive Analysis Webapp / Redusic / MuseReduce /</title>
+    <link type="text/css" rel="stylesheet" href="css/vendor/normalize.css">
+    <link type="text/css" rel="stylesheet" href="css/vendor/jBox.all.min.css">
+    <link type="text/css" rel="stylesheet" href="css/vendor/select2.min.css">
+    <link type="text/css" rel="stylesheet" href="css/vendor/style.css">
+    <link type="text/css" rel="stylesheet" href="css/app.css">
+    <script type="text/javascript" src="/js/vendor/jquery-3.1.1.min.js" defer></script>
+    <script type="text/javascript" src="/js/vendor/verovio-toolkit.js" defer></script>
+    <script type="text/javascript" src="/js/app.js" defer></script>
```

Comment:
The first slash in front of the script paths should be removed, thus:

```js
<script type="text/javascript" src="js/vendor/jquery-3.1.1.min.js" defer></script>
<script type="text/javascript" src="js/vendor/verovio-toolkit.js" defer></script>
<script type="text/javascript" src="js/app.js" defer></script>
```

The scripts may fail to load otherwise.

Reply:
That’s fixed. From what I see it only happened when you directly opened `index.html` in `file:///` or using `npm run start` (which is already available before of this PR).

- I remove the leading `/`.
- I’ve also updated the BrowserSync config for `npm run start` users.